Jenkinsfile

```diff
@@ -1,14 +1,23 @@
-#!/usr/bin/env groovy
 @Library('jenkins_lib')_
 
 pipeline {
-  agent any
+  //agent {label 'slave'}
+  agent {label 'nebula-slave03'}
```

Comment:
As discussed with Chandresh, we are keeping this hard coded slave for now until devOps finalizes the right strategy to pick the right slave based on our component dependencies. We will create a separate US to track this further once they fix their part.

Reply:
US to track 
https://guavus-jira.atlassian.net/browse/UIC-1111